tests/fixtures/insufficient_metrics_ema_test.json

```diff
@@ -0,0 +1 @@
+{"resource_usage":[{"executors":[{"allocated":[{"name":"cpus","role":"*","scalar":{"value":1.1},"type":"SCALAR"},{"name":"mem","role":"*","scalar":{"value":160},"type":"SCALAR"}],"executor_info":{"command":{"shell":true,"value":"/home/skonefal/usr/libexec/mesos/mesos-executor"},"executor_id":{"value":"serenity2"},"framework_id":{"value":"20150612-182512-16842879-5050-6428-0001"},"name":"Command Executor (Task: serenity2) (Command: sh -c 'watch pwd')","resources":[{"name":"cpus","role":"*","scalar":{"value":0.1},"type":"SCALAR"},{"name":"mem","role":"*","scalar":{"value":32},"type":"SCALAR"}],"source":"serenity2"},"statistics":{"cpus_limit":1.1,"cpus_user_time_secs":0.07,"mem_limit_bytes":167772160,"mem_rss_bytes":70057984,"timestamp":1434126327.18446}},{"allocated":[{"name":"cpus","role":"*","scalar":{"value":1.1},"type":"SCALAR"},{"name":"mem","role":"*","scalar":{"value":160},"type":"SCALAR"}],"executor_info":{"command":{"shell":true,"value":"/home/skonefal/usr/libexec/mesos/mesos-executor"},"executor_id":{"value":"test1"},"framework_id":{"value":"20150612-182512-16842879-5050-6428-0000"},"name":"Command Executor (Task: test1) (Command: sh -c 'watch ls')","resources":[{"name":"cpus","role":"*","scalar":{"value":0.1},"type":"SCALAR"},{"name":"mem","role":"*","scalar":{"value":32},"type":"SCALAR"}],"source":"test1"},"statistics":{"cpus_limit":1.1,"cpus_user_time_secs":0.1,"mem_limit_bytes":167772160,"mem_rss_bytes":71229440,"timestamp":1434126327.20772}}]},{"executors":[{"allocated":[{"name":"cpus","role":"*","scalar":{"value":1.1},"type":"SCALAR"},{"name":"mem","role":"*","scalar":{"value":160},"type":"SCALAR"}],"executor_info":{"command":{"shell":true,"value":"/home/skonefal/usr/libexec/mesos/mesos-executor"},"executor_id":{"value":"serenity2"},"framework_id":{"value":"20150612-182512-16842879-5050-6428-0001"},"name":"Command Executor (Task: serenity2) (Command: sh -c 'watch pwd')","resources":[{"name":"cpus","role":"*","scalar":{"value":0.1},"type":"SCALAR"},{"name":"mem","role":"*","scalar":{"value":32},"type":"SCALAR"}],"source":"serenity2"},"statistics":{"cpus_limit":1.1,"mem_limit_bytes":167772160,"mem_rss_bytes":70057984,"timestamp":1434126327.18446}},{"allocated":[{"name":"cpus","role":"*","scalar":{"value":1.1},"type":"SCALAR"},{"name":"mem","role":"*","scalar":{"value":160},"type":"SCALAR"}],"executor_info":{"command":{"shell":true,"value":"/home/skonefal/usr/libexec/mesos/mesos-executor"},"executor_id":{"value":"test1"},"framework_id":{"value":"20150612-182512-16842879-5050-6428-0000"},"name":"Command Executor (Task: test1) (Command: sh -c 'watch ls')","resources":[{"name":"cpus","role":"*","scalar":{"value":0.1},"type":"SCALAR"},{"name":"mem","role":"*","scalar":{"value":32},"type":"SCALAR"}],"source":"test1"}}]}]}
```

Comment:
I think you should add additional folder eg. /ema inside fixtures.
Soon we will be searching for a needle in a haystack.